hoi/metrics/info_tot.py

```diff
@@ -91,24 +86,24 @@ def fit(self, minsize=2, maxsize=None, method='gcmi', **kwargs):
         # _______________________________ HOI _________________________________
 
         # get progress bar
-        pbar = get_pbar(
-            iterable=range(minsize, maxsize + 1), leave=False,
-        )
+        pbar = get_pbar(iterable=range(minsize, maxsize + 1), leave=False)
 
         # prepare the shapes of outputs
-        n_mults = sum([ccomb(self.n_features - 1, c) for c in range(
-            minsize, maxsize + 1)])
+        # E501
+        x = self.n_features
```

Comment:
The code will break with this change because the variable `x` already contains the data. I'm surprised that the error is that the line is too long...